[13515] Vulcanexus supported platforms (#10)
eProsima/vulcanexus: docs/rst/platforms/platforms.rst

.. _platforms:

Supported platforms
===================

Vulcanexus distribution is officially available in the platforms specified in the table below.

.. list-table:: Vulcanexus officially supported platforms
    :header-rows: 1

    * - Vulcanexus Version
      - Architecture
      - OS

>     * - Galactic Gamble

Reviewer: We can add a link here in the same way we do in the releases section

      - amd64
      - Ubuntu Focal (20.04)

However, Vulcanexus is an extended ROS 2 distribution which is officially supported in the platforms stated in the `REP 2000 specification <https://www.ros.org/reps/rep-2000.html>`_.

> Thus, building Vulcanexus distribution for these platforms is probably going to be successful.

Reviewer: ```suggestion
Thus, building Vulcanexus distribution for these platforms is expected to succed.
```

> Any other platform not mentioned in the REP 2000 specification could also be used and have a successful built.

Reviewer: ```suggestion
Other platforms not mentioned in the REP 2000 specification may also build successfully and be used.
```
